.taskcluster.yml

```diff
@@ -0,0 +1,31 @@
+version: 0
+metadata:
+  name: servo
+  description: >-
+    A modern, high-performance browser engine designed for both application
+    and embedded use.
+  owner: '{{ event.head.user.email }}'
+  source: '{{ event.head.repo.url }}'
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events: []
+    payload:
+      maxRunTime: 3600
+      image: servobrowser/servo-linux-dev
+      command:
+        - /bin/bash
+        - '--login'
+        - '-c'
+        - >-
+          git clone {{event.head.repo.url}} servo && cd servo && git config
+          advice.detachedHead false && git checkout {{event.head.sha}} &&
+          etc/ci/taskcluster-test.sh
```

Comment:
Would be nice if each command was on a separate line, or at the least no command was split across multiple lines.